experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -316,16 +321,24 @@ export class FetchInstrumentation extends InstrumentationBase<
 
         function endSpanOnSuccess(span: api.Span, response: Response) {
           plugin._applyAttributesAfterFetch(span, options, response);
+          const spanResponse = {
```

Comment:
Was that meant to be used as a third argument of `_endSpan` call? Right now the `spanResponse` variable is unused.

Reply:
Sorry, I did not see your comment on this PR. Looks like this is some leftover from me trying to figure out what eventually turned out to be caused by #2884. I unintentionally added this to commit [f6d0b0f](https://github.com/open-telemetry/opentelemetry-js/pull/2871/commits/f6d0b0f93d6defee412b7e359466ac3950672684).